ci/before-install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
+  openssl aes-256-cbc -K $encrypted_c25aa352ba20_key -iv $encrypted_c25aa352ba20_iv
```

Comment:
I take it `$encrypted_c25aa352ba20_key` / `$encrypted_c25aa352ba20_iv` are injected into the build environment somehow?